packages/@lwc/integration-karma/test/mixed-shadow-mode/restrictions/x/component/component.js

```diff
@@ -1,8 +1,6 @@
 import { LightningElement, api } from 'lwc';
 
 export default class extends LightningElement {
-    static shadowSupportMode = 'any';
```

Comment:
@nolanlawson could you take a look at this change? 

Wanted to make sure this test still behaves as intended.

Reply:
We had a hard time understanding this test! We decided to remove this in the end, because we realized that it was basically a NOOP due to the feature flag not being enabled.